### ISSUE TITLE
Define which field types `Timestamp (or similar) Column` field in Get Rows Polling Trigger should work with

### ISSUE THREAD

**Issue by stas-fomenko** [elasticio/jdbc-component#24]
DESCRIPTION: 
Currently, this field does not support `datetime` field type.

**What needs to be done:**

**1.** 
The requirement by PO is needed: which field types of:
- mssql
- postgre
- mysql
- oracle
 should be supported by `Timestamp (or similar) Column` field

**2.**
Proper updates should be done about the updated requirements: investigating, development, testing, etc.

**Comment by shulkaolka** [comment]
As described [there](http://db.apache.org/ojb/docu/guides/jdbc-types.html), JDBC has 3 generic sql types similar to timestamp:
```
DATE | java.sql.Date
TIME | java.sql.Time
TIMESTAMP | java.sql.Timestamp
```
As implementation of this task there are 2 solutions:
1. Create a new provider for field `pollingField` and show there only fields with these three types
2. Describe in README.md what exactly fields customer should select from the existing list.
I prefer the first one.
@stas-fomenko @A3a3e1 your thoughts?

**Comment by shulkaolka** [comment]
- [x] All the branches if the repo should be checked (merged/deleted if possible) before starting development
- [x] Development team has all needed requirements
- [x] Access to the needed service with all needed permissions is granted
- [x] No open tasks that could block Component development process
- [x] The E.io’s GitHub repository must be checked if there is another implementation of the component under development. If it exists, new repository should not be created
- [x] Component architecture should be discussed and approved before start coding

**Development:**

- [x] Code is written and formatted in accordance with the Styling guide
- [x] Code (incl. configs, test samples, environment variables etc.) MUST NOT contain any sensitive data like passwords, tokens, API keys etc.
- [x] Component logs does not contain any sensitive data (credentials, input/output requests, Environment variables)
- [ ] Code is covered with unit tests
- [x] Component code is covered with integration tests (optional)
- [ ] Documentation (README.md file in the Github) is 100% ready
- [ ] Changelogs (CHANGELOG.md) are created/updated
- [ ] package.json or build.gradle is updated:
  - Component's version is updated according to Semantic Versioning 2.0.0
  - Component should be dockerised, add: "buildType":"docker"
- [ ] CI is set up
- [ ] All development branches are reviewed

**After QA:**

- [ ] All development branches are merged into master branch
- [ ] Sprint review preparation is finished (demo flow is built)

**QA:**

- [ ] Test cases for the component are ready at the TestRail
- [ ] Functional testing is finished
- [ ] Regression testing is finished (optional)
- [ ] Component is deployed to the production stage (app.elastic.io) and/or other required stage
  - Component has a proper access level
  - Required Environment Variables are set
  - Smoke test is finished
- [ ] Release notes is done
- [ ] Component Roadmap is updated
- [ ] Component spreadsheet is updated
